server/backend/api/app/jwt_validation.py

```diff
@@ -238,7 +238,11 @@ def get_access_roles(claims: dict = Depends(authorize)):
     return groups
 
 
-def get_request_username(claims: dict = Depends(authorize)):
-    requester = claims[COGNITO_USERNAME_KEY]
-    LOGGER.debug(f"Current requester for API: {requester}")
-    return requester
+def get_request_cognito_username(claims: dict = Depends(authorize)):
```

Comment:
Why not call this cognito_user_id to avoid confusion.

Reply:
:) Thanks, I actually prefer and will change it to be called "**cognito_user_id**".
It was only because the fact its name on Cognito attribute mapping is "**username**" so I refactored it from previous method name "get_request_username" to be more meaningful, but do prefer to call it "cognito_user_id".
Will change.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/69f76097-4b4b-49b0-8ad8-bf948552a72b)

Reply:
Changed (get_request_cognito_user_id)